.babelrc

```diff
@@ -1,19 +1,20 @@
 {
```

Comment:
Not directly related, but this file must have changed from `babel.json` to `.babelrc` at some point. Dockerfile still has references to babel.json, which breaks the build. 🤔